src/baldr/incident_singleton.h

```diff
@@ -190,11 +202,12 @@ struct incident_singleton_t {
     }
 
     // some setup for continuous operation
-    bool first_run = true;
+    size_t run_count = 0;
     time_t last_scan = 0;
-    size_t latent_count = 0;
     time_t max_loading_latency =
         config.get<time_t>("incident_max_loading_latency", DEFAULT_MAX_LOADING_LATENCY);
+    std::unordered_set<uint64_t> seen;
+    seen.reserve(tileset.size());
```

Comment:
when we do a scan, either of the changelog or the directory, if we track what tiles we see in the tileset. if any of them disappeared (from the changelog or from the directory) then we want to also remove them from the cache